pkg/providers/instancetype/nodetemplate_test.go

```diff
@@ -529,22 +588,22 @@ var _ = Describe("NodeTemplate/InstanceTypes", func() {
 			// Should pack onto same instance
 			coretest.UnschedulablePod(coretest.PodOptions{
 				ResourceRequirements: v1.ResourceRequirements{
-					Requests: v1.ResourceList{v1alpha1.ResourceAWSNeuron: resource.MustParse("2")},
-					Limits:   v1.ResourceList{v1alpha1.ResourceAWSNeuron: resource.MustParse("2")},
+					Requests: v1.ResourceList{v1alpha1.ResourceAWSNeuron: resource.MustParse("8")},
+					Limits:   v1.ResourceList{v1alpha1.ResourceAWSNeuron: resource.MustParse("8")},
 				},
 			}),
 			// Should pack onto a separate instance
 			coretest.UnschedulablePod(coretest.PodOptions{
 				ResourceRequirements: v1.ResourceRequirements{
-					Requests: v1.ResourceList{v1alpha1.ResourceAWSNeuron: resource.MustParse("4")},
-					Limits:   v1.ResourceList{v1alpha1.ResourceAWSNeuron: resource.MustParse("4")},
+					Requests: v1.ResourceList{v1alpha1.ResourceAWSNeuron: resource.MustParse("16")},
```

Comment:
I didn't look deeply at this but why did you multiply all of these by 4? Looks like these are still requesting the neuron device so I'm not sure that this should have changed, should it?

Reply:
Since `inf1.24xlarge` was not originally being offered in these tests it would pack these pods into multiple `inf1.6xlarge` instances instead of a single `inf1.24xlarge` iirc. By upping the requests we will instead pack into multiple `inf1.24xlarge` instances.